.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
 name: Bug
-description: Some user-visible feature works incorrectly
+description: Some feature works incorrectly
```

Comment:
That's still implied, but now it is less weird to users: we should not convince users that their problems are user-visible, and, hopefully, we would not forget that